code_examples/sdk_examples/src/core_features/linking/index.ts

```diff
@@ -1,12 +1,10 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
 import type { KeyringPair } from '@polkadot/keyring/types'
 
 import { randomUUID } from 'crypto'
 
-import * as Kilt from '@kiltprotocol/sdk-js'
-
 import { claimWeb3Name } from '../web3names/01_claim'
 import { createSimpleFullDid } from '../did/04_full_did_simple'
```

Comment:
Why this change here?

Reply:
But the logic so far, as far as I can see, is to import
1. types
2. external packages
3. KILT SDK
4. Internal modules/files

Reply:
Ok, I will change back.